scripts/install/audiobookshelf.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Audiobookshelf installer
+
+_install_audiobookshelf() {
+    echo_progress_start "Installing audiobookshelf apt sources"
```

Comment:
`echo_progress_start` kind of expects an `echo_progress_done`, think of the two as the slices of bread in a waiting sandwich